keycloak-radius-spi/Makefile

```diff
@@ -0,0 +1,22 @@
+# default build target
+all::
+
+all:: dev
+.PHONY: build dev reset
+
+WORKING_DIR := $(shell pwd)
+
+build:
+	docker run -it --rm -v $(WORKING_DIR):/usr/local/app -w /usr/local/app maven:3-openjdk-11 mvn clean && mvn package
```

Comment:
@adaybujeda can you help me to understand this?
- Why are you running `mvn clean` inside the `maven` container, but running `mvn package` in the host OS?
- What is the advantage to running the build inside the container if the host OS still requires Maven and Java?
- Couldn't it introduce problems if the version of Maven on the host OS is not the same as the one in the container?